### PR TITLE
Initial add of dev package

### PR DESCRIPTION
**Pull request by jcftang**
This package is needed for tensorflow

### CODE REVIEW

File: var/spack/repos/builtin/py-dev/package.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+
+
+class PyDev(PythonPackage):
+    """libraries and tools for Python development"""
+
+    homepage = "https://pypi.python.org/pypi/dev"
+    url      = "https://pypi.python.org/packages/53/34/e0d4da6c3e9ea8fdcc4657699f2ca62d5c4ac18763a897feb690c2fb0574/dev-0.4.0.tar.gz"
```
Comment (adamjstewart) [inline var/spack/repos/builtin/py-dev/package.py]: Please use https://pypi.io/packages/source/d/dev/dev-0.4.0.tar.gz instead. That way we can download other versions from the same URL.

**Comment by jcftang** [comment]
@adamjstewart thats updated it was an oversight on my part.

**Comment by adamjstewart** [comment]
No worries, it happens all the time. PyPI URLs are just plain confusing.

**Comment by alalazo** [comment]
@jcftang @adamjstewart Am I missing something or this package should be one folder below in the hierarchy?  Currently it is at the same level as the `package` folder...

**Comment by adamjstewart** [comment]
Ha, you're right.

**Comment by tgamblin** [comment]
`py-dev` should be one level deeper in the packages directory.  @jcftang: were you actually able to `spack install` this and see it in `spack list`?

**Pull request merged**